unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -61,7 +61,7 @@ export const VerificationStatus = ({ config, onVerified, onClose }: Props) => {
   const { isLoading: isKeyLoading, data: key } = useQuery(
     [network, tokenId, lockAddress],
     async () => {
-      if (lockVersion && lockVersion >= 10) {
+      if (lockVersion && lockVersion >= 10 && tokenId) {
```

Comment:
Some older QR codes might have been
```suggestion
      // Some older QR codes might have been generated without a tokenId in the payload. Clean up after January 2023
      if (lockVersion && lockVersion >= 10 && tokenId) {
```